src/math_functions.cpp

```diff
@@ -674,6 +674,35 @@ double maxwell_spectrum(double T) {
 }
 
 
+double gaussian_spectrum(double mean, double standard_deviation) {
+  double r2 = 1.01;
+  double v1 = 0.;
+  double v2 = 0.;
+  
+  // perhaps there should be a limit to the number of resamples
+  while ( r2 > 1 ) {
+    v1 = 2 * prn() - 1.;
+    v2 = 2 * prn() - 1.;
+    double r = std::pow(v1,2) + std::pow(v2,2);
+    r2 = std::pow(r,2);
+    if ( r2 < 1 ) {
```

Comment:
For consistency with existing C++ code, I'd remove leading/trailing spaces in the condition for an `if` statement:
```C++
if (r2 < 1) {
```